gui

```diff
@@ -67,7 +67,7 @@ fi
 
 #if this script runs itself with a 3rd argument, then yad used it.
 if [ ! -z "$3" ];then
-  input="$4" #the 4th argument is the "sysname" column - invisible and used internally for scripts. MAY BE EMPTY, but the 3rd argument is guaranteed to not be empty.
+  input="$3" #the 3rd argument is the "sysname" column - invisible and used internally for scripts.
```

Comment:
no botspot, I think `input` needs to stay as `input="$4"`
you didn't change the number of arguments output

Reply:
> no botspot, I think `input` needs to stay as `input="$4"` you didn't change the number of arguments output

On my system at least, yad avoids outputting any "special" color columns. So arguments passed to `gui` will be one word to the left.

Reply:
on my system, with this change, all submenus break.
removing `--print-column=4` is what stopped the columns from showing up